dubbo-remoting/dubbo-remoting-http3/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyHttp3ConnectionClient.java

```diff
@@ -81,17 +81,15 @@ protected void initChannel(NioDatagramChannel ch) {
         datagramChannel.set(nettyDatagramChannel);
         nettyDatagramChannel.closeFuture().addListener(channelFuture -> datagramChannel.set(null));
 
-        int heartbeat = UrlUtils.getHeartbeat(getUrl());
         NettyConnectionHandler connectionHandler = new NettyConnectionHandler(this);
         bootstrap = QuicChannel.newBootstrap(nettyDatagramChannel)
                 .handler(new ChannelInitializer<QuicChannel>() {
                     @Override
                     protected void initChannel(QuicChannel ch) {
-                        ch.pipeline()
-                                .addLast(new IdleStateHandler(heartbeat, 0, 0, MILLISECONDS))
-                                .addLast(Constants.CONNECTION_HANDLER_NAME, connectionHandler)
-                                .addLast(new Http3ClientConnectionHandler());
-
+                        ChannelPipeline pipeline = ch.pipeline();
+                        pipeline.addLast(new Http3ClientConnectionHandler());
+                        pipeline.addLast(Constants.CONNECTION_HANDLER_NAME, connectionHandler);
+                        pipelineConfigurator.accept(pipeline);
```

Comment:
Please ensure that it is not null for safety.

Reply:
Checking in constructor.